<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgUserSelect</class>
 <widget class="QDialog" name="DlgUserSelect">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>294</width>
    <height>193</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>成员属性</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#DlgUserSelect {
	background: rgb(255,255,255);
}
QListWidget {
	border: 0px;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QListWidget" name="userListWidget"/>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
